packages/template-retail-react-app/jest-setup.js

```diff
@@ -73,7 +73,13 @@ afterEach(() => {
     global.server.resetHandlers()
 })
 afterAll(() => {
-    global.server.close()
+    // Intentionally not closing the server!
+    // We run into many race condition issues,
+    // that was cause by the server close too soon
+    // and the tests not well written in an proper async manner.
+    // Let's not close the server and see how things goes.
+    // We can revisit this.
+    // global.server.close()
```

Comment:
Any consequences of n ot closing the server here? Will it be terminated after the job is done? I am wondering if it would drain the CI resource?

Reply:
so far we haven't seen any side effect